rholang/src/main/scala/coop/rchain/rholang/interpreter/accounting/Costs.scala

```diff
@@ -78,4 +79,6 @@ trait Costs {
   ) {
     def storageCost: Cost = Cost(a.map(a => gm.toByteArray(a).size).sum)
   }
+
+  final val MAX_VALUE = PhloLimit(Long.MaxValue)
```

Comment:
this is the same as the one in casper/src/test/scala/coop/rchain/casper/util/Costs.scala

Reply:
The other one is gone now and this is the only `MAX_VALUE`.